### DISCUSSION TITLE
Welcome to verinfast Discussions!

### DISCUSSION THREAD

**Discussion by aylusltd** [VerinFast/verinfast#168] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by bcdady** [comment]
I see a few open issues from 2023. Do maintainers pick up issues reported by community users and prioritize them in GitHub, or is there another (private?) tool used for issue feedback, bugs, and feature requests?
